### PR TITLE
<WIP> Update build tools to clang-8

### PR DESCRIPTION
**Pull request by BRMcLaren**
Signed-off-by: Brett McLaren <brmclare@microsoft.com>

### CODE REVIEW

**Review by achamayou — APPROVED** [review]

**Review by radhikaj — APPROVED** [review]

**Comment by BRMcLaren** [comment]
https://github.com/openenclave/openenclave/pull/3580/files#r494100233

Pulling in changes requested, as I merged the PRs.

**Comment by BRMcLaren** [comment]
https://oe-jenkins-tf.westeurope.cloudapp.azure.com/blue/organizations/jenkins/CI-CD_Infrastructure%2FOpenEnclave-E2E-Testing/detail/OpenEnclave-E2E-Testing/255/pipeline

Running.

**Comment by BRMcLaren** [comment]
https://oe-jenkins-tf.westeurope.cloudapp.azure.com/job/CI-CD_Infrastructure/job/OpenEnclave-E2E-Testing/256/

trying again with ansible fixes

**Comment by BRMcLaren** [comment]
Code failures witnessed here #3615 are blocking this pr.

**Comment by BRMcLaren** [comment]
bors try

File: scripts/ansible/roles/linux/openenclave/tasks/ubuntu/lvi-setup.yml [hunk 1]
```diff
@@ -13,13 +13,13 @@
...
-  shell: "which clang-7"
+  shell: "which clang-8"
   changed_when: false
   register: clang
   ignore_errors: yes
 
-- name: Detect currently installed clang++-7
-  shell: "which clang++-7"
+- name: Detect currently installed clang++-8
+  shell: "which clang++-8"
```
Comment (BRMcLaren) [inline scripts/ansible/roles/linux/openenclave/tasks/ubuntu/lvi-setup.yml:22]: .

**Comment by anakrish** [comment]
@BRMcLaren https://github.com/openenclave/openenclave/pull/3780 should have the fix for tests/cppException failure.

**Comment by BRMcLaren** [comment]
bors try

**Pull request closed without merging**